tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -48,21 +46,32 @@ class KatelloCertsCheckTestCase(TestCase):
     CA cert (a.k.a cacert.crt or rootCA.pem) can be used as bundle file.
     """
 
-    @classmethod
-    def setUpClass(cls):
+    @pytest.fixture(scope="module")
+    def cert_data(self):
         """Get host name, scripts, and create working directory."""
-        super(KatelloCertsCheckTestCase, cls).setUpClass()
-        _, cls.sat6_hostname = os.path.split(settings.server.hostname)
-        cls.capsule_hostname = 'capsule.example.com'
-        cls.key_file_name = '{0}/{0}.key'.format(cls.sat6_hostname)
-        cls.cert_file_name = '{0}/{0}.crt'.format(cls.sat6_hostname)
-        cls.ca_bundle_file_name = 'cacert.crt'
-        cls.SUCCESS_MSG = "Validation succeeded"
+        _, sat6_hostname = os.path.split(settings.server.hostname)
```

Comment:
Could we change it to `sat6_hostname = settings.server.hostname` ? I don't think os.path.split is required here.